dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java

```diff
@@ -1065,6 +1075,56 @@ public boolean canEdit(Context context, Item item) throws SQLException {
         return collectionService.canEditBoolean(context, item.getOwningCollection(), false);
     }
 
+    /**
+     * Finds all Indexed Items where the current user has submit rights. If the user is an Admin,
+     * this is all Indexed Items. Otherwise, it includes those Items where
+     * an indexed "submit" policy lists either the eperson or one of the eperson's groups
+     *
+     * @param context                    DSpace context
+     * @param discoverQuery
+     * @return                           discovery search result objects
+     * @throws SQLException              if something goes wrong
+     * @throws SearchServiceException    if search error
+     */
+    private DiscoverResult retrieveItemsWithEdit(Context context, DiscoverQuery discoverQuery, String q)
```

Comment:
Tiny thing: Please add the `String q` param to the JavaDocs & describe it.

Reply:
Thanks for the feedback! Added the doc.